clients/gtest/blas1_gtest.yaml

```diff
@@ -21,70 +25,192 @@ Definitions:
-    - copy:  *single_double_precisions_complex_real
-    - dot:   *double_precision_complex_real
-    - dotc:  *double_precision_complex_real
-    - scal:  *single_double_precisions_complex_real
-    - scal:  *single_double_complex_real_in_complex_out
-    - swap:  *single_double_precisions_complex_real
-    - rot:   *rot_precisions
-    - rotm:   *single_double_precisions_complex_real
-
-- name: blas1_bad_arg
-  category: pre_checkin
-  function:
-    - nrm2_bad_arg:  *single_double_precisions_complex_real
-    - asum_bad_arg:  *single_double_precisions_complex_real
-    - iamax_bad_arg: *single_double_precisions_complex_real
-    - iamin_bad_arg: *single_double_precisions_complex_real
-    - axpy_bad_arg:  *half_single_precisions_complex_real
-    - copy_bad_arg:  *single_double_precisions_complex_real
-    - dot_bad_arg:   *half_bfloat_single_double_complex_real_precisions
-    - dotc_bad_arg:  *single_double_precisions_complex
-    - scal_bad_arg:  *single_double_precisions_complex_real
-    - scal_bad_arg:  *single_double_complex_real_in_complex_out
-    - swap_bad_arg:  *single_double_precisions_complex_real
-    - rot_bad_arg:   *rot_precisions
-    - rotg_bad_arg:  *rotg_precisions
-    - rotm_bad_arg:  *single_double_precisions_complex_real
-    - rotmg_bad_arg: *single_double_precisions_complex_real
-
-- name: blas1
-  category: quick
-  function:
-    - rotg:  *rotg_precisions
-    - rotmg: *single_double_precisions_complex_real
+  - name: blas1
+    category: quick
+    N: [ -1, 0, 5, 10, 500, 1000, 1024, 1025, 7111, 10000, 33792 ]
+    incx_incy: *incx_incy_range
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    function:
+      - nrm2:  *single_double_precisions_complex_real
+      - asum:  *single_double_precisions_complex_real
+      - iamax: *single_double_precisions_complex_real
+  #   - iamin: *single_double_precisions_complex_real # broken for now -- cause unknown
+      - axpy:  *half_single_precisions_complex_real
+      - copy:  *single_double_precisions_complex_real
+      - dot:   *half_bfloat_single_double_complex_real_precisions
+      - dotc:  *single_double_precisions_complex
+      - scal:  *single_double_precisions_complex_real
+      - scal:  *single_double_complex_real_in_complex_out
+      - swap:  *single_double_precisions_complex_real
+      - rot:   *rot_precisions
+      - rotm:   *single_double_precisions_complex_real
+
+  - name: blas1_batched
+    category: quick
+    N: [ -1, 0, 5, 33792 ]
+    incx_incy: *incx_incy_range
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [-1, 5]
+    function:
+      - scal_batched: *single_double_precisions_complex_real
+      - scal_batched: *single_double_complex_real_in_complex_out
+
+  - name: blas1_strided_batched
+    category: quick
+    N: [ -1, 0, 5, 33792 ]
+    incx_incy: *incx_incy_range
+    alpha_beta: *alpha_beta_range
+    alphai_betai: *alphai_betai_range
+    batch_count: [-1, 5]
```

Comment:
You may want to add batch_count of 0 to the quick tests to test the API quick return functionality.

Reply:
Good call, added.